src/components/scene/ar-hit-test.js

```diff
@@ -366,6 +366,11 @@ module.exports.Component = register('ar-hit-test', {
     this.makeBBox();
   },
   update: function () {
+    // If it is disabled it's cleaned up
+    if (this.data.enabled === false) {
+      this.hitTest = null;
+      this.bboxMesh.visible = false;
+    }
```

Comment:
Do we have to restore those if the component is reenabled?

Reply:
They get reassigned when an xr selectstart happens 

the mesh is made visible  
and the hit-test set to the one corresponding to that input